packages/mockotlpserver/lib/cli.js

```diff
@@ -186,6 +199,10 @@ async function main() {
             case 'logs-summary':
                 printers.push(new LogsSummaryPrinter(log));
                 break;
+
+            case 'trace-file':
+                printers.push(new FilePrinter(log));
+                break;
```

Comment:
Should the `trace-file` "printer" be added the `printers` automatically if `--ui` is specified?

Reply:
One may want to open the UI just to inspect traces from past sessions but I think you question points to the case someone starting the UI but forgetting to put the printer in place, therefore not being able to se traces in the new traces UI. I think is as good point, I'll add it in the next commit